src/MetaMorpho.sol

```diff
@@ -528,6 +541,10 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
     }
 
     /// @inheritdoc IERC4626
+    /// @dev Warning: Can revert. This goes against EIP4626. If a market is broken on Morpho `_supplyBalance` can revert
+    /// as well as `totalAssets`. The allocator can always remove the broken market from `withdrawQueue` if the
+    /// situation
+    /// persists and unlock the vault.
```

Comment:
```suggestion
    /// @dev Warning: May revert when virtually accruing interest on a market of the withdraw queue, due to the market's IRM. In such case, funds supplied to the market are lost as long as accruing interest reverts and the allocator role can remove the market from `withdrawQueue` to realize the loss and unlock the vault.
```

1. I don't like the naming "broken" markets because it is too vague
2. If we're adding this natspec to this function, we should add it to all functions that depend on it?

Reply:
1. agree
2. Not possible because some are in the ERC4626 implementation

Reply:
Should we still duplicate it for all functions that we override and that depend on `totalAssets`? I fear this comment gets lost because the integrator doesn't connect the dots

Reply:
Better to replicate, than nothing I'd say so yes let's do that.

Reply:
I added the comment on entry points and getters but not internal functions. I think this is the most important places